### PR TITLE
create ballot papers with pdfmake (fixes #2298)

### LINKED ISSUE #2298: make ballot papers with pdfmake
By @ThomasJunk  from #2288 

> In order to completely substitute reportlab, ballot papers have to be generated via pdfMake.
I scribbled together a quick version at the playground of pdfMake.
To enhance, the borderstyle of the used table has to be a little tweaked.

[https://github.com/OpenSlides/OpenSlides/files/428136/stimmzettel.txt](url)

Ballot paper shall contain the Motion number, the title of the motion, options for "yes", "no" "abstention".
An A4 Page shall contain 6 "section" with bullet papers.

We also need the circle somehow

### PR DESCRIPTION
**Pull request by tsiegleauq**
This creates the Ballot Papers using PDFmake as originally requested by @ThomasJunk in the assigned issue.

@emanuelschuetze 

Example here:
[Motion 3 Ballot.pdf](https://github.com/OpenSlides/OpenSlides/files/441385/Motion.3.Ballot.pdf)
(german title would be "Antrag 3 Wahlgang.pdf")

I changed the usage of the word "Nummer" since we cannot translate it on the fly using angular gettext currently. therefore I did chose the international "#" for number

### CODE REVIEW

**Comment by normanjaeckel** [comment]
Ok. Not tested.

Please do not forget to cleanup (delete PDF on server side, move PDF code to extra js file, ...)

**Comment by tsiegleauq** [comment]
> Please do not forget to cleanup (delete PDF on server side, move PDF code to extra js file, ...)

There is an [issue for that](https://github.com/OpenSlides/OpenSlides/issues/2307).
I have no information about any priorities...

**Comment by emanuelschuetze** [comment]
As discussed:
* new (real) circle (unicode or base64 image)
* no "#"
* left align
* thinner line/border

**Comment by tsiegleauq** [comment]
@emanuelschuetze 
I made the required changes, can be seen here:
[Antrag 2 Wahlgang.pdf](https://github.com/OpenSlides/OpenSlides/files/445094/Antrag.2.Wahlgang.pdf)

I did also refractor and comment the code a lot.
pdfmake supports the drawing of simple forms btw, such as circles. I created a simple function for.
Additionally to the required changes, I thought it would be a good idea to color the cutting-lines gray

File: openslides/motions/static/js/motions/site.js [hunk 1]
```diff
@@ -859,6 +991,15 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions', 'OpenSlid
           });
         };
 
+        //make PDF for polls
+        $scope.makePollPDF = function() {
+            var id = motion.identifier,
+                title = motion.getTitle($scope.version),
+                filename = gettextCatalog.getString("Motion") + " " + id + " " + gettextCatalog.getString("Ballot") + ".pdf",
```
Comment (emanuelschuetze) [inline openslides/motions/static/js/motions/site.js]: Please prevent blanks in filename. Use e.g. Motion1-ballot.pdf

**Comment by emanuelschuetze** [comment]
Looks good. One remark. Then please rebase again.

**Comment by tsiegleauq** [comment]
> Looks good. One remark. Then please rebase again.

I also changed the filename of the other .download function, since it used the space character as well.

**Comment by tsiegleauq** [comment]
@emanuelschuetze 
commits should be clean now.

**Comment by emanuelschuetze** [comment]
Ok. looks good. Ready for last review and merge.

**Pull request merged**